NuclearData/ceNeutronData/aceDatabase/aceNeutronDatabase_class.f90

```diff
@@ -912,27 +925,35 @@ subroutine init(self, dict, ptr, silent )
       mat => mm_getMatPtr(i)
 
       ! Load nuclide indices on storage space
-      ! Find if material is fissile
+      ! Find if material is fissile and if stochastic
+      ! mixing temperature bounds are respected
       isFissileMat = .false.
       ! Loop over nuclides
       do j = 1, size(mat % nuclides)
-        name = trim(mat % nuclides(j) % toChar())
-
-        if (mat % nuclides(j) % hasSab) then
-          zaid = trim(mat % nuclides(j) % toChar())
-          file = trim(mat % nuclides(j) % file_Sab)
-          name = zaid // '+' // file
-          deallocate(zaid, file)
-        end if
-
+        name = self % makeNuclideName(mat % nuclides(j))
```

Comment:
We could move this procedure inside materialMenu, and save the name inside nuclideInfo, so that we don't need to repeat this several times

Reply:
The only problem with this is transferring the Sab discrimination info, i.e., the symbols used to staple things together. They could become universal variables, but feels unnecessary. If there is a case for using it in more than one place though I would be behind it.

Reply:
I just proposed this because this function is already called twice! No big deal, but maybe it would be neater.